phpstan.neon

```diff
@@ -30,14 +30,6 @@ parameters:
         - '#^Call to an undefined method DateInterval::(spec|optimize)\(\)\.$#'
         - '#^Property Carbon\\Carbon::\$timezone \(Carbon\\CarbonTimeZone\) does not accept string\.$#'
         - '#^Method class@anonymous/tests/Carbon/TestingAidsTest\.php:\d+::modify\(\) should return class@anonymous/tests/Carbon/TestingAidsTest\.php:\d+ but returns \(?DateTimeImmutable#'
-        -
-            message: '#^Undefined variable: \$this$#'
-            paths:
-                - src/Carbon/Traits/Mixin.php
-        -
-            message: '#^Variable \$this in isset\(\) is never defined\.$#'
-            paths:
-                - src/Carbon/Traits/Mixin.php
```

Comment:
This is always good.

Reply:
Depends, if it disapear by hacking $this into ${'this'} then I would say no, because it's just fooling the tool and adding complexity/execution time while not actually changing what Psalm is complaining about.